torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -934,6 +934,7 @@ class ShapePropagator {
             "aten::sub(Tensor self, Tensor other, *, Scalar alpha) -> Tensor",
             "aten::mul(Tensor self, Tensor other) -> Tensor",
             "aten::div(Tensor self, Tensor other) -> Tensor",
+            "aten::div(Tensor self, Tensor other, str rounding_mode) -> Tensor",
```

Comment:
Do we need to change this to pass a test? If not, I suggest we don't edit it. It's not well tested.